### ISSUE TITLE
PDF scrolling is hard to follow

### ISSUE THREAD

**Issue by ricconi** [duckduckgo/iOS#422]
When I open a multi-page pdf file and try to scroll down, there is a glitch that seems to show content from several pages away, then quickly scrolls back to where it should be.  It seems to happen whenever I change scrolling direction (for example, scroll down a few pages, lift off finger, then scroll up), so trying to quickly find a part of a document is hard.  There also seems to be a similar glitch when zooming in.

**Comment by bwaresiak** [comment]
This is now tracked internally at https://app.asana.com/0/413877547933097/1119805176098508

**Comment by bwaresiak** [comment]
Thank you for your report, @ricconi 

To be honest, I couldn't reproduce this issue while reading PDF documents (be it short ones, long ones, with images, etc). I've seen small glitch around navigation bar when I deliberately scroll just a tiny bit upwards, but for regular usage I could not find what you've described. 

Could you share some details about the device you are using (model, iOS version)? Also, please try if the issue persists on the latest DuckDuckGo app release. Finally, if you can give us example of PDFs that break for you, or even make a screenshot/video showing the problem, that would be great.

Thank you!

**Comment by bwaresiak** [comment]
Since there is no further feedback and we couldn't reproduce this, I'm closing this issue.
